manila/share/manager.py

```diff
@@ -5400,6 +5437,8 @@ def share_server_migration_check(self, context, share_server_id, dest_host,
             # the validations.
             driver_result['compatible'] = False
 
+        self.driver.network_api.delete_port_bindings(
```

Comment:
because the migration-check and migration-start are separated steps, triggered by clients, they don't necessarily follow each other. The network allocations will be created again at the beginning of the migration-start step. In fact, the compatibility check is also done there again. I think recreating the network allocations builds a consistent pattern.

Reply:
I see, thanks for explaining - I wasn't aware that the migration check is a separate step